Dockerfile

```
@@ -19,10 +24,54 @@ RUN yum -y install \
     freetype-devel \
     libart_lgpl-devel \
     gcc \
-    groff
+    groff \
+    wget \
+    automake \
+    libffi-devel \
+    libyaml-devel
+
+# RUN cd tmp && \
+#     wget https://www.openssl.org/source/openssl-1.0.2a.tar.gz && \
+#     tar -zxvf openssl-*.tar.gz && \
+#     cd openssl-*/ && \
+#     ./config -fpic shared && make && make install && \
+#     echo "/usr/local/ssl/lib" >> /etc/ld.so.conf && \
+#     ldconfig
+
+RUN yum -y install \
+    make \
+    automake \
+    autoconf \
+    install \
+    perl-ExtUtils-MakeMaker \
+    fakeroot
+
+RUN curl -o /tmp/openssl-1.0.2u.tar.gz http://artfiles.org/openssl.org/source/old/1.0.2/openssl-1.0.2u.tar.gz && \
+    cd /tmp && tar -xzf /tmp/openssl-1.0.2u.tar.gz && \
+    cd /tmp/openssl-1.0.2u && ./Configure linux-x86_64 no-shared --openssldir=/opt/openssl -fPIC && make && make install && \
+    echo "/usr/local/ssl/lib" >> /etc/ld.so.conf && ldconfig && \
+    cd - && rm -rf /tmp/openssl-1.0.2u && rm /tmp/openssl-1.0.2u.tar.gz
+
+COPY ./curl-7.46.0.tar.gz /tmp/curl-7.46.0.tar.gz
+
+RUN cd /tmp && tar -xzf /tmp/curl-7.46.0.tar.gz && \
+    cd /tmp/curl-7.46.0 && LIBS="-ldl" ./configure --enable-static --prefix=/opt/curl --with-ssl=/opt/openssl && make all && make install && \
```

Missing `&&` but this should probably be line-broken too
```suggestion
    cd /tmp/curl-7.46.0 && LIBS="-ldl" && \
    ./configure --enable-static --prefix=/opt/curl --with-ssl=/opt/openssl && \
    make all && \
    make install && \
```